### PR TITLE
fix race condition

### PR DESCRIPTION
**Pull request by shogo82148**
goroutine concurrency expertへの道は長いですね。

go1.9beta1で、-race オプションを有効にした時のテスト結果です。長いのでgistに
https://gist.github.com/shogo82148/25097134b34e9d5276225cc9a72eb37a

### CODE REVIEW

**Comment by shogo82148** [comment]
ping

**Comment by mackee** [comment]
I review now. Just a moment.

**Comment by shogo82148** [comment]
happy birthday :tada:

File: Makefile [hunk 1]
```diff
@@ -6,7 +6,8 @@ cmd/ekbo/ekbo: *.go cmd/ekbo/main.go
 .PHONY: clean install get-deps test packages
 
 test:
-	go test
+	go test -v -race
```
Comment (mackee) [inline Makefile]: 👍

File: config.go [hunk 2]
```diff
@@ -17,6 +18,8 @@ type Config struct {
 	Endpoint        string            `yaml:"endpoint"`
 	StrictBroadcast bool              `yaml:"strict_broadcast"`
 	ProxySetHeader  map[string]string `yaml:"proxy_set_header"`
+	SendTimeout     time.Duration     `yaml:"send_timeout"`
```
Comment (mackee) [inline config.go]: Must be set default value(ex. `"1s"`) for backward compatibility.
time.Duration's zero value is 0s.

**Pull request merged**